templates/admin/misc/icon_list.html.twig

```diff
@@ -66,126 +71,136 @@
-    {% for icon in twemoji %}
-        {% set iconPath = icon|replace({(webRoot): ''}) %}
-        {#
-            Any icon with basename that has a dash will be considered as a variant to avoid recurisvely searching for "parent" icon.
-            It happens that all icon that have variants have at least a prefix of 4-5 characters.
-        #}
-        {% if '-' in iconPath and iconPath|basename|split('-')[0]|length > 3 %}
+        {% if ('-' in iconPath and iconPath|basename|split('-')[0]|length > 3) and (type =='twemoji') %}
             <div class="icon variant">
                 {{ icon | pimcore_twemoji_variant_icon | raw }}
                 <div class="label">{{ iconPath|basename }}</div>
             </div>
         {% else %}
             <div class="icon">
-                {{ icon | pimcore_inline_icon | raw }}
-                <div class="label">{{ iconPath|basename }}</div>
+
+                {% if type == 'flags' %}
+                    {% set country = iconPath|basename|replace({'.svg': ''}) %}
+                    <div class="country_name" data-isocode="{{ country }}"></div>
+                {% endif %}
+
+                {% if loop.index < 300 %}
+                    {{ icon | pimcore_inline_icon | raw }}
+                {% else %}
+                    {{ icon | pimcore_lazy_icon | raw }}
+                {% endif %}
+
+                <div class="label">
+                    {{ iconPath in iconsCss  ? '*' : '' }}
+                    {{ iconPath|basename }}
+                </div>
             </div>
         {% endif %}
     {% endfor %}
 </div>
 
-<div class="info">
-    Flags
-</div>
-
-<table>
-    <tr>
-        <th>Flag</th>
-        <th>Code</th>
-        <th>Name</th>
-    </tr>
-    {% for langOpt in languageOptions %}
-        <tr>
-            <td class="language-icon">{{ langOpt['flag'] | pimcore_inline_icon | raw }}</td>
-            <td>{{ langOpt['language'] }}</td>
-            <td>{{ langOpt['display'] }}</td>
-        </tr>
-    {% endfor %}
-</table>
-
 <script
-    src="https://code.jquery.com/jquery-3.7.0.slim.js"
-    integrity="sha256-7GO+jepT9gJe9LB4XFf8snVOjX3iYNb0FHYr5LI1N5c="
-    crossorigin="anonymous"
-    {{ pimcore_csp.getNonceHtmlAttribute()|raw }}></script>
-<script src="/bundles/pimcoreadmin/js/pimcore/common.js"></script>
-<script src="/bundles/pimcoreadmin/js/pimcore/functions.js"></script>
-<script src="/bundles/pimcoreadmin/js/pimcore/helpers.js"></script>
+    src="https://code.jquery.com/jquery-3.7.1.min.js"
+    integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo="
+    crossorigin="anonymous"></script>
 
 <script {{ pimcore_csp.getNonceHtmlAttribute()|raw }}>
     $( document ).ready(function() {
+
         // Add click event to copy icon path on all icons
-        $('img').each(function () {
-            $(this).click(function () {
-                pimcore.helpers.copyStringToClipboard($(this).data('imgpath'));
+        $('img, div.lazy-load').each(function () {
```

Comment:
Just because I read about the `.on('click')` topic. I guess the whole loop could be even replaced with something like this, right? The advantage would be that you have only one event listener instead of one for each icon which could be massively more lightweight. ;)